zebedee-cms/src/main/java/com/github/onsdigital/zebedee/json/publishing/PublishedCollection.java

```diff
@@ -12,8 +12,8 @@
 
 public class PublishedCollection extends CollectionBase {
 
-    private CollectionDataset datasets;
-    private CollectionDatasetVersion datasetVersions;
+    private CollectionDataset[] datasets;
+    private CollectionDatasetVersion[] datasetVersions;
```

Comment:
I think these should be initialised in the constructor - to empty array. To prevent null pointer.

Reply:
Change to list `List<CollectionDataset>` and `List<CollectionDatasetVersion>`